data/fintype.lean

```diff
@@ -242,26 +242,26 @@ instance (α : Type u) (β : Type v) [fintype α] [fintype β] : fintype (α ⊕
   fintype.card (α ⊕ β) = fintype.card α + fintype.card β :=
 by rw [sum.fintype, fintype.of_equiv_card]; simp
 
-lemma fintype.card_le_of_injective [fintype α] [fintype β] (f : α → β) 
+lemma fintype.card_le_of_injective [fintype α] [fintype β] (f : α → β)
   (hf : function.injective f) : fintype.card α ≤ fintype.card β :=
 by haveI := classical.prop_decidable; exact
 finset.card_le_card_of_inj_on f (λ _ _, finset.mem_univ _) (λ _ _ _ _ h, hf h)
 
 lemma fintype.card_eq_one_iff [fintype α] : fintype.card α = 1 ↔ (∃ x : α, ∀ y, y = x) :=
-by rw [← fintype.card_unit, fintype.card_eq]; 
-exact ⟨λ ⟨a⟩, ⟨a.symm unit.star, λ y, a.bijective.1 (subsingleton.elim _ _)⟩, 
-λ ⟨x, hx⟩, ⟨⟨λ _, unit.star, λ _, x, λ _, (hx _).trans (hx _).symm, 
+by rw [← fintype.card_unit, fintype.card_eq]; exact
+⟨λ ⟨a⟩, ⟨a.symm unit.star, λ y, a.bijective.1 (subsingleton.elim _ _)⟩,
+  λ ⟨x, hx⟩, ⟨⟨λ _, unit.star, λ _, x, λ _, (hx _).trans (hx _).symm,
     λ _, subsingleton.elim _ _⟩⟩⟩
```

Comment:
With this indentation, it is not clear what the last lambda belongs to I would align the lambdas as follows:

```
⟨λ ⟨a⟩, ⟨a.symm unit.star, λ y, a.bijective.1 (subsingleton.elim _ _)⟩,
 λ ⟨x, hx⟩, ⟨⟨λ _, unit.star, 
            λ _, x, λ _, (hx _).trans (hx _).symm,
            λ _, subsingleton.elim _ _⟩⟩⟩
```

Reply:
I'm okay with this indentation - it's not really an attempt to align anything, it's just a generic "continuation" indent.

Reply:
I understand but with all the brackets, I find it hard to parse visually.

Reply:
maybe just no wrapping? It's not that long

Reply:
That could also work